doc/classes/MeshConvexDecompositionSettings.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="MeshConvexDecompositionSettings" inherits="RefCounted" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Parameters to be used with a [Mesh] convex decomposition operation.
+	</brief_description>
+	<description>
+		Parameters to be used with a [Mesh] convex decomposition operation.
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="convexhull_approximation" type="bool" setter="set_convexhull_approximation" getter="get_convexhull_approximation" default="true">
+			If enabled uses approximation for computing convex hulls.
+		</member>
+		<member name="convexhull_downsampling" type="int" setter="set_convexhull_downsampling" getter="get_convexhull_downsampling" default="4">
```

Comment:
"convexhull" should be 2 words, me thinks. Also please use `[code]` around numeric literals in the descriptions.

Reply:
I have no opinion if this should be 1 or 2 words, it is a 1:1 copy on how the variables were named before. I did a quick search and there are cases throughout the engine for both namings so if we want to unify them we properly should do all in one sweep.

Reply:
From what I see, in this same exposed API you mostly have it as two words. So IMO this particular case should be addressed in this PR. If there are any other cases in public names that don't follow that convention, we probably can't do much about it at this point (backwards compatibility and all). But this API here is new, and is inconsistent between itself.